src/app/pages/home/home.js

```diff
@@ -52,16 +52,14 @@ export default class Home extends BaseView {
             return;
         }
 
-        let secondaryNavHeight = appView.header.secondaryNavHeight;
+        let headerHeight = appView.header.height;
 
-        if (window.pageYOffset > secondaryNavHeight && !appView.header.isPinned()) {
-            let height = appView.header.height;
-
-            appView.header.reset().collapse().pin();
-            this.el.style.paddingTop = `${height / 10}rem`;
-        } else if (window.pageYOffset <= secondaryNavHeight && !appView.header.isTransparent()) {
-            appView.header.reset().transparent();
-            this.el.style.paddingTop = '0';
+        if (window.pageYOffset >= headerHeight + 100) {
+            appView.header.pin().visible();
+        } else if (window.pageYOffset < headerHeight + 100 && window.pageYOffset >= headerHeight + 20) {
```

Comment:
@Fredasaurus indicated to me yesterday that his intent was for the header to scroll as it currently does on https://os-webview-dev.openstax.org (so that it transitions smoothly as soon as you scroll 1px beyond it).  I don't think the `+ 100` and `+ 20` are thus applicable/desirable.